src/components/OrgNode.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <li :id="data.user_id" :class="'org-node' + ( data.user_id === $store.state.profilePreview.userId ? ' org-node--current' : '')">
-    <a :href="`/profile/${data.user_id}`" @click.prevent="previewProfile(data.user_id)" ref="openProfileTrigger">
+  <li :id="data.user_id" :class="'org-node' + ( data.user_id === this.$route.params.userId ? ' org-node--current' : '')">
+    <router-link :to="{ name: 'Orgchart', params: { userId: data.user_id } }">
```

Comment:
Looks like we're losing the `ref` here; it is used to store which link opened the org chart preview, so that we can return focus to it when the preview is closed. This may be as simple ad adding the `ref` to `router-link`?

Reply:
I solved it slightly different. How do you feel about it?